torch/_inductor/lowering.py

```diff
@@ -5855,6 +5853,74 @@ def create_nn_parameter(self, placeholder):
     return TensorBox.create(ir.BindNNParameter(self, placeholder))
 
 
+@register_lowering(torch.ops.aten.resize)
+def resize(x, size, *, memory_format=None):
+    if memory_format is None:
+        memory_format = torch.contiguous_format
+    if memory_format == torch.preserve_format:
+        raise RuntimeError(f"unsupported memory format: {memory_format}")
+
+    assert isinstance(x, TensorBox)
+    assert isinstance(size, (list, tuple))
+
+    size = V.graph.sizevars.size_hints(size)
```

Comment:
This is going to raise on unbacked symints. We should probably fallback in that case. Mind adding a test?

Reply:
Actually, do you really need this line?

Reply:
You are right here we don't actually need this.